src/firmware/app/control/trajectory_planner_test.cpp

```diff
@@ -962,63 +423,50 @@ TEST_F(TrajectoryPlannerTest, test_get_constant_time_interpolation_curved_line)
         .y = {.coefficients = {1, 0, 1, 0}},
 
     };
+    Polynomial1dOrder3_t orientation_profile = {.coefficients = {0, 0, 0, 0}
 
-    FirmwareRobotPathParameters_t path_parameters;
-    path_parameters.path                       = path;
-    path_parameters.t_start                    = 0;
-    path_parameters.t_end                      = 1;
-    path_parameters.num_segments               = 500;
-    path_parameters.max_allowable_acceleration = 3;
-    path_parameters.max_allowable_speed        = 3;
-    path_parameters.initial_speed              = 0;
-    path_parameters.final_speed                = 0;
-
-    PositionTrajectoryElement_t const_arc_elements[TRAJECTORY_PLANNER_MAX_NUM_ELEMENTS];
-    float speed_profile[TRAJECTORY_PLANNER_MAX_NUM_ELEMENTS];
-    PositionTrajectory_t trajectory = {.trajectory_elements = const_arc_elements,
-                                       .path_parameters     = path_parameters,
-                                       .speed_profile       = speed_profile};
-
+    };
 
-    TrajectoryPlannerGenerationStatus_t status =
-        app_trajectory_planner_generateConstantArcLengthPositionTrajectory(&trajectory);
-    EXPECT_EQ(OK, status);
+    FirmwareRobotPathParameters_t path_parameters;
+    path_parameters.path                              = path;
+    path_parameters.t_start                           = 0;
+    path_parameters.t_end                             = 1;
+    path_parameters.num_segments                      = 500;
+    path_parameters.max_allowable_linear_acceleration = 3;
+    path_parameters.max_allowable_linear_speed        = 3;
+    path_parameters.initial_linear_speed              = 0;
+    path_parameters.final_linear_speed                = 0;
 
-    // Create the parameterization to contain the desired number of segments
-    CREATE_STATIC_ARC_LENGTH_PARAMETRIZATION(arc_length_param,
-                                             TRAJECTORY_PLANNER_MAX_NUM_ELEMENTS);
+    PositionTrajectoryElement_t const_param_elements[TRAJECTORY_PLANNER_MAX_NUM_ELEMENTS];
+    PositionTrajectory_t trajectory = {.trajectory_elements = const_param_elements,
+                                       .path_parameters     = path_parameters};
 
-    // Get all of the points for the arc length parameterization (Not constant arc length
-    // segments)
-    shared_polynomial_getArcLengthParametrizationOrder3(
-        path, path_parameters.t_start, path_parameters.t_end, arc_length_param);
+    TrajectoryPlannerGenerationStatus_t status_variable_time =
+        app_trajectory_planner_generateConstantParameterizationPositionTrajectory(
+            &trajectory);
 
-    PositionTrajectory_t const_interp_trajectory;
+    EXPECT_EQ(status_variable_time, OK);
```

Comment:
bit of a nit, but `status_variable_time` doesn't  seem super descriptive here? Just confused about where `variable_time` is coming from. You can write this like:
``` suggestion
    {
        TrajectoryPlannerGenerationStatus_t status =
            app_trajectory_planner_generateConstantParameterizationPositionTrajectory(
                &trajectory);

        EXPECT_EQ(status, OK);
    }
```
if you're concerned about variable scope. Same goes for below.

Reply:
Changes to use a single status variable.